cpu/Makefile.include.msp430_common

```diff
@@ -14,4 +14,41 @@ export ASFLAGS += $(CFLAGS_CPU) --defsym $(CPU_MODEL)=1 $(CFLAGS_DBG)
 export LINKFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG) $(CFLAGS_OPT) -Wl,--gc-sections -static -lgcc
 
 # Import all toolchain settings
-include $(RIOTCPU)/Makefile.include.gnu
+include $(RIOTCPU)/Makefile.include.$(TOOLCHAIN)
+
+# Test that the compiler exists, try to adjust PREFIX otherwise
+ifeq (,$(shell sh -c 'command -v $(CC)'))
+export PREFIX:=$(PREFIX)elf-
+endif
+
+ifeq (,$(MSP430_USE_NEWLIB))
+# MSPGCC (the old compiler) is usually used with msp-libc, while the new
+# upstream GCC msp430 port (sometimes referred to as the Red Hat MSP430
+# compiler) is usually accompanied by newlib.
+# The new msp430 port started with version 4.9.x, the latest release of the old
+# mspgcc is 4.6.3 from the beginning of 2012 and doesn't look like it will receive
```

Comment:
Is there any reason why we would want to support a compiler which has not been updated for 6 years and most likely won't receive any fix ever again?